trans/static/js/common.js

```diff
@@ -53,34 +53,20 @@ function sendPrintJob(print_task_url) {
     });
 }
 
-// Added by Emil Abbasov IOI2019
-function validateFinalizeTranslation(form, n) {
-	
-	check = document.getElementById('not_translating');
-	
-	if(check.checked) {
-		return confirm('Are you sure?');
-	}
-	
-    if(document.getElementsByName('reopen').length!= n) {
-        alert('All tasks must be finalized before this action!');
-        return false;
+function validateFinalizeTranslation(contest_slug, tasks_length) {
+    const frozen_tasks = document.querySelectorAll(
```

Comment:
Nit: indentation looks off? (Mix between tabs and spaces in this file)

Reply:
Done.